pkg/util/mocks/env/env.go

```diff
@@ -407,6 +421,20 @@ func (mr *MockInterfaceMockRecorder) FeatureIsSet(arg0 interface{}) *gomock.Call
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "FeatureIsSet", reflect.TypeOf((*MockInterface)(nil).FeatureIsSet), arg0)
 }
 
+// GenevaLoggingEnvironment mocks base method.
+func (m *MockInterface) GenevaLoggingEnvironment() string {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "GenevaLoggingEnvironment")
+	ret0, _ := ret[0].(string)
+	return ret0
+}
+
+// GenevaLoggingEnvironment indicates an expected call of GenevaLoggingEnvironment.
+func (mr *MockInterfaceMockRecorder) GenevaLoggingEnvironment() *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "GenevaLoggingEnvironment", reflect.TypeOf((*MockInterface)(nil).GenevaLoggingEnvironment))
+}
+
```

Comment:
@troy0820 - "[ ] Update unit test to include new (generation of new mocks have been made and existing unit tests do not cover newly included config params, etc)"

Reply:
From what I can tell, existing unit tests do not cover anything related to mdsd logging. I agree that unit tests should be added, but feel that creating a brand new suite of tests for mdsd logging is beyond the scope of this PR/work item.